tests/schema/tests.py

```diff
@@ -1673,3 +1673,20 @@ def test_alter_field_default_dropped(self):
             )
             if connection.features.can_introspect_default:
                 self.assertIsNone(field.default)
+
+    def test_add_textfield_unhashable_default(self):
```

Comment:
Tests seem to have some redundant ops.

Looks like this is enough to reproduce the issue, no need to create model instance and set `get_db_prep_save` (we are not going to save any objects.

```python
    def test_add_textfield_unhashable_default(self):
        # Create the table
        with connection.schema_editor() as editor:
            editor.create_model(Author)

        # Create a field that has an unhashable default
        new_field = TextField(default={})
        new_field.set_attributes_from_name("info")

        # Fail if an exception is raised
        with connection.schema_editor() as editor:
            editor.add_field(Author, new_field)
```